### ISSUE TITLE
1.0 Release

### ISSUE THREAD

**Issue by itsazzad** [MLstate/PEPS#55]
May I know how far is it from the very much expected `1.0 Release` as the possibility to be released was in the last February?

**Comment by hbbio** [comment]
Yes, we're late behind schedule, mostly because (if you look at the [roadmap](https://github.com/MLstate/PEPS/wiki/Roadmap)) there is one feature yet which we haven't started: Generalize Drag and Drop. We are mostly busy with customer requests for now, but will do our best to release before summer.
